components/LoginPage.vue

```diff
@@ -9,19 +9,40 @@
         <form @submit.prevent="login">
           <div class="form-group">
             <label for="email">Silliman Email</label>
-            <input type="email" id="email" v-model="email" class="form-control" placeholder="" required />
+            <input
+              type="email"
+              id="email"
+              v-model="email"
+              class="form-control"
+              placeholder=""
+              required
+            />
           </div>
 
           <div class="form-group">
             <label for="password">Password</label>
             <div class="password-input-wrapper">
-              <input :type="passwordInputType" id="password" v-model="password" class="form-control" placeholder="" required />
-              <div class="eye-icon" v-if="password.length > 0" @click="togglePassword">
+              <input
+                :type="passwordInputType"
+                id="password"
+                v-model="password"
+                class="form-control"
+                placeholder=""
+                required
+              />
+              <div
+                class="eye-icon"
+                v-if="password.length > 0"
```

Comment:
no logics in template, use computed()
```suggestion
                v-if="password.length > 0"
```

Reply:
goods man